core/tests/protractor_desktop/coreEditorAndPlayerFeatures.js

```diff
@@ -33,6 +33,148 @@ var ExplorationEditorPage =
 var ExplorationPlayerPage =
   require('../protractor_utils/ExplorationPlayerPage.js');
 var LibraryPage = require('../protractor_utils/LibraryPage.js');
+var ExplorationEditorPage =
+  require('../protractor_utils/ExplorationEditorPage.js');
+
+describe('Enable correctness feedback and set correctness', function() {
+  var explorationEditorPage = null;
+  var explorationEditorMainTab = null;
+  var explorationEditorSettingsTab = null;
+  var explorationPlayerPage = null;
+  var currentExplorationIndex = 0;
+  var EXPLORATION_TITLE = 'Dummy Exploration';
+  var explorationTitle = null;
+  var libraryPage = null;
+  var correctOptions = [
+    ['MultipleChoiceInput', 'Correct!'],
+    ['TextInput', 'One'],
+    ['NumericInput', 3]
+  ];
+
+
+  var applyCommonSettings = function() {
```

Comment:
Should we move this to the beforeEach section after creating an exploration? I think this will make the tests more readable.

(If required we can explicitly call `enableCorrectnessFeedback` whenever required in the test.)

Reply:
I think moving it to the before Each is a good idea..